app/bundles/FormBundle/Controller/FieldController.php

```diff
@@ -155,6 +160,7 @@ public function newAction()
                     'field'         => $formField,
                     'id'            => $keyId,
                     'formId'        => $formId,
+                    'formName'      => $formEntity->generateFormName(),
```

Comment:
If new form $formEntity is null.
`'formName'      => $formEntity !== null ? $formEntity->generateFormName() : ""`

Reply:
The string cannot be empty. I got route errors if it were and that's why I added the formName here.

Reply:
The problem is that $formEntity is null if you create a new form.

Reply:
Good catch. Thanks!